mdtools/_metadata.py

```diff
@@ -40,6 +40,10 @@
 __author__ = "Andreas Thum, Len Kimms"
 # Remove possible dublicates and ensure alphabetical order
 author_list = sorted(set(__author__.split(", ")), key=str.lower)
```

Comment:
Very very pedantic: I can get sorted in front of you by simply adding a space before my name (since ' ' < 'a'). Such white space issues are hard to spot, especially if names are similar so that the alphabetic order is not obvious.
I proppose: `author_list = sorted(set(map(str.strip, __author__.split(","))), key=str.lower)`

Reply:
Good suggestion!